### ISSUE TITLE
Annotation track keeps disappearing when I scroll around

### ISSUE THREAD

**Issue by AlexdeMendoza** [igvteam/igv-webapp#260]
Dear IGV team, 

I have some sessions on IGV app and I host my files on Globus (a cloud storage used by many universities). I didn't use to have any problem, but this week I realised that the annotation file keeps not loading any genes when I scroll around or change genomic position. Just when I open my session I do see the genes, but then they disappear and I can't do anything about it. Even if I load the local file (sorted gff with index done with IGVtools), it happens the same. Since this wasn't happening before, I suspect this might be on your end. 

Thanks, 

A.

**Comment by AlexdeMendoza** [comment]
Yes, this is igv.org/app, it works well in the local version. I can share a link to the session if you like, but since this is unpublished data I wouldn't like to have this accessible for too long.

**Comment by helgathorv** [comment]
You can share the session link to the IGV Team privately by sending us an email to igv-team@broadinstitute.org

**Comment by AlexdeMendoza** [comment]
Sure, thanks, I have emailed (although I got back an automatic response saying I shouldn't).

**Comment by helgathorv** [comment]
Thanks - I should have mentioned the auto-reply - we encourage people to use the igv-help forum or GitHub issues to contact us about problems. But you already did that!

**Comment by helgathorv** [comment]
I do see the problem with the scrolling - and we'll investigate. But in the meantime, IGV does show the data if you explicitly enter the genomic coordinates in the search box in the toolbar. The coordinates are also updated if you scroll, so if you scroll past the point where the data disappears, a quick workaround is to click on the magnifying glass next to the genomics coordinates and IGV will update the tracks to show the data.

**Comment by AlexdeMendoza** [comment]
hey, thanks!
Yes, I do the zooming in and out to refresh the tracks, but this time it
doesn't fix it, I really don't know why as this same session used to work.
Just to recap, I used IGVtools to sort and index the annotation gtf track
so it loaded quicker, so it could be that. In contrast, the RepeatMasker
bed file, which has no index, is seen perfectly well.

A.

**Comment by AlexdeMendoza** [comment]
ah, and I have another question, from the IGV app browser I can get nice
svg files that I can then modify in Adobe Illlustrator. However, the latest
version of IGV desktop, produces some svgs that cannot be opened in Mac or
Illustrator, so not sure about the incompatibility.

Thanks,

A.

**Comment by helgathorv** [comment]
Can you open new issue in the IGV desktop repo about that: https://github.com/igvteam/igv/issues
You say that it produces "some svgs that cannot be opened". So do you mean that not all the svgs it produces have that problem? In the new issue, please include the steps to create a file that has the problem.

**Comment by jrobinso** [comment]
Thanks for the test data,  I will look into it.  I suspect its the index.  For a more performant index, that is also recognized by other tools,  I recommend. tabix, part of the sam/bam/htslib suite of tools.

RE SVG, I see you opened another issue.  Implementations for desktop and igv.js differ completely,  desktop goes through the Java "swing" interface, igv.js is a more direct export.  I can't really comment further without more details, but I see you opened an issue on this.  We should discuss it there.
